tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -1303,24 +1303,25 @@ def test_spillover(self, cloud_storage_type):
 
```

Comment:
> rptest: actual e2e throttling test

would you mind adding detail to the commit message? what is "actual ... test" mean?